lib/portage/package/ebuild/doebuild.py

```diff
@@ -2050,6 +2050,12 @@ def _postinst_bsdflags(mysettings):
 			(_shell_quote(mysettings["ROOT"]),
 			_shell_quote(os.path.join(mysettings["T"], "bsdflags.mtree"))))
 
+def _postinst_maintainer_nag(mysettings, maintainers):
```

Comment:
Needs to be guarded (not necessarily here) by FEATURES, e.g. maintainer-needed-nag (thanks Zac).